### DISCUSSION TITLE
Different gene scores of Different TileMat Size?

### DISCUSSION THREAD

**Discussion by AritaZ-hang** [GreenleafLab/ArchR#1578] [Questions / Documentation]
Hello maintainers of ArchR, I use `addGeneScoreMatrix`this function at the very beginning of creating arrow files, and I wonder whether the change of tile size(`TileMatParams`, actually) in `createArrowFiles`will affect the distribution of gene score matrix. I've read the manual and it seems that `addGeneScoreMatrix` uses another tile size parameter independent from that in`addTileMatrix`. However in our data, a certain group of genes has different ranges of gene scores in two set of tile sizes(`TileMatParams`), not imputed, so I think it's not because of iterative LSI.

Relevant codes are below. Thanks for your help in advance.

```R
ArrowFiles = createArrowFiles(inputFiles = file.list, 
                             TileMatParams = list(tileSize = 500), # another is 50, for our data is a from a small genome species,
                             GeneScoreMatParams = list（tileSize = 2000), addTileMat = TRUE, addGeneScoreMat = TRUE)
```

**Comment by rcorces** [comment]
I dont see how this would be the case - `TileMatParams` shouldnt affect anything with the `GeneScoreMatrix` as far as I can tell. Make sure you have set a seed in your analysis as there may be some randomized operations that you arent accounting for.
